src/index.js

```diff
@@ -22,7 +22,45 @@ export default class ToggleBlock {
     this.wrapper = undefined;
   }
 
-  render() {
+  onKeyDown(paragraphId, e) {
+    try {
```

Comment:
@juanhurtado10 I think it would be nice, it would give more clarity to what the method does and give more context for when it should be called. So I'm going to apply the required changes.